README.md

```diff
@@ -427,4 +427,29 @@ Example usage
         github-token: ${{ steps.get-token.outputs.token }}
         threshold: ${{ inputs.inclint-threshold }}
 
-```
\ No newline at end of file
+```
+
+## JavaScript
+
+### LTS-lint
+
+This action runs [prettier](https://prettier.io) and [eslint](https://eslint.org/)
+on the target repository.
+It is currently used exclusively in the _onprem platform_ by the LTS team.
+This action does not provide any configuration for the linters,
+it relies on a configfile (such as `package.json`) located in the target repository,
+making it as generic as possible.
```

Comment:
Could you point to which repositories, group of repositories it is recommended to use that action?

Reply:
I see your point - I was talking more about those lts specific repositories where this will/should be used. Unfortunately, other teams are not interested in sharing such linter rules ([I talked with them on ui-sync](https://piwikpro.atlassian.net/wiki/spaces/PPMS/pages/86116302849/2023-02-20+UI+Sync#Key-takeaways)). So I would

Reply:
As for LTS team, we are only using this action in the `Promil-platform-onprem` repository. I am fine with not merging this PR, if it doesn't bring value for other teams. But the reason this PR exists is because other teams **were interested** in a common JS linting action during our Sprint Review, so I am confused now.

Reply:
@mmieszczak do you remember which teams?

Reply:
Z tego co pamiętam to `Promil` i `This Is Fine`

Reply:
Ok, so let's merge it and see how adoption will go.

Reply:
Thanks!